<launch>
  <node pkg="move_arm" type="move_arm" name="move_arm" output="screen">    
   <param name="ik_service_name" value="pr2_ik_node/ik_service" />     
   <param name="ik_query_name" value="pr2_ik_node/ik_query" />  
  </node>
</launch>
